server/controllers/likes.controllers.js

```diff
@@ -0,0 +1,64 @@
+import model from '../models';
+import likeHelper from '../helpers/likes.helpers';
+import validations from '../helpers/validations';
+
+const { Article, User } = model;
+const articleLike = {
+  async toggleLike(req, res) {
+    const { articleId } = req.params;
+    const { userId } = req.body;
+    try {
+      // validate article and user Id
+      if (
+        !validations.verifyUUID(articleId) ||
+        !validations.verifyUUID(userId)
+      ) {
+        res.status(404).json({
+          status: 404,
+          message: 'Id does not exist',
+        });
```

Comment:
Should this be returning 404? Are we not verifying that the id is valid/invalid? If it's invalid you can return a bad request status code